src/sections/LoginAuth0.js

```diff
@@ -49,16 +59,32 @@ class LoginAuth0 extends Component {
         // authResult.expiresIn - string with the access token's expiration time in seconds
         // authResult.idToken - ID token JWT containing user profile information
 
+        const { name, email, picture } = authResult.idTokenPayload;
+
         this.props
           .authenticate({
             variables: {
-              accessToken: authResult.accessToken
+              accessToken: authResult.accessToken,
+              name,
+              email,
+              picture
             }
           })
           .then(({ data }) => {
             const userInfo = data.authenticateUser;
             this.props.setAuthSession(authResult, userInfo.id, userInfo.token);
             this.setState({ isLoggingIn: false });
+          })
+          .catch(error => {
+            const alertMessage = error.message.includes(
```

Comment:
We should have a polyfill for this for [users using internet explorer](https://caniuse.com/#search=includes). It doesn't look like [our version of CreateReactApp polyfills it](https://github.com/codefordenver/members/blob/master/docs/CreateReactApp.md#supported-language-features-and-polyfills), and, while I thought we had the babel-polyfill included, I don't see it.